dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
Your custom module loader should be fine as the browser will block while downloading/executing the JS listed in that comment. The original require behaviour is restored at the end.

```
<script>
  // Save and delete module, forcing cross-storage to populate global
  var saveModule = module;
  delete module;
</script>
<script src="https://url/for/cross-storage-client.js"></script>
<script>
  // restore module, restoring CommonJS module functionality
  module = saveModule;
</script>
```